webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue

```diff
@@ -165,18 +166,22 @@ export default {
       this.loadingPeople = true;
       // Using 'limitToFetch + 1' to retrieve current user and then delete it from result
       // to finally let only 'limitToFetch' users
+      if (this.abortController) {
+        this.abortController.abort();
+      }
+      this.abortController = new AbortController();
       let searchUsersFunction;
       if (this.filter === 'connections') {
-        searchUsersFunction = this.$userService.getConnections(this.keyword, this.offset, this.limitToFetch + 1, this.fieldsToRetrieve);
+        searchUsersFunction = this.$userService.getConnections(this.keyword, this.offset, 200 + 1, this.fieldsToRetrieve, this.abortController.signal);
```

Comment:
```suggestion
        searchUsersFunction = this.$userService.getConnections(this.keyword, this.offset, 200 + 1, this.fieldsToRetrieve, this.abortController.signal);
```
Why did you hardcode the limitToFetch ?

Reply:
restored!